dev/TEAL_opcodes.md

```diff
@@ -554,9 +557,20 @@ See `bnz` for details on how branches work. `b` always jumps to the offset.
 
 params: account index, application id (top of the stack on opcode entry)
 
-## app_local_get
+## app_local_gets
 
 - Opcode: 0x62
+- Pops: *... stack*, {uint64 A}, {[]byte B}
+- Pushes: any
+- read from account's A from local state of the current application key B  => value
```

Comment:
maybe
"read from account specified by Txn.Account[A] ..."

Reply:
done for this and all other